documents/satzung/gebuehrenordnung.tex

```diff
@@ -1,12 +1,20 @@
-\documentclass[fontsize=12pt, paper=a4, DIV14, parskip]{scrreprt}
+\documentclass[
+	fontsize=12pt,
+	paper=a4,
+	DIV14,
+	parskip,
+]{scrreprt}
+
+\usepackage{fontspec}
+\setmainfont{Source Sans Pro}
+\renewcommand\familydefault\sfdefault
 
 \usepackage[ngerman]{babel}
 \usepackage[utf8]{inputenc}
```

Comment:
When using fontspec / lualatex, eurosym, inputenc and fontenc should not be used.

Reply:
Do you have a quick fix?

Reply:
just remove

Reply:
removed